discord/utils.py

```diff
@@ -1276,6 +1278,14 @@ def format_dt(dt: datetime.datetime, /, style: TimestampStyle | None = None) ->
     :class:`str`
         The formatted string.
     """
+    if isinstance(dt, datetime.time):
+        if style is None:
+            style = "t"
+        elif style not in ("t", "T"):
+            raise ValueError("Time styles must be 't' or 'T'.")
+        dt = datetime.datetime.combine(
+            datetime.datetime.fromtimestamp(DISCORD_EPOCH, tz=datetime.UTC), dt
```

Comment:
can't you use `.now`?

Reply:
Yeah I can just thought it would be more consistent if it was always the same.

Reply:
I think if we used `now`, it would also be reasonable to allow other time styles (and just have it default to today)

Reply:
Alright

Reply:
Yes, default to today, not to the minimum date, that leads to strange results.